datadog_checks_dev/datadog_checks/dev/tooling/create.py

```diff
@@ -111,6 +111,7 @@ def construct_template_fields(integration_name, repo_choice, integration_type, *
 
     config = {
         'author': author,
+        'auto_install': 'false' if repo_choice == 'marketplace' or integration_type == 'metrics_pull' else 'true',
```

Comment:
that's probably a dumb question but in your previous commit it was a boolean in the manifest but here it will be a string now I think, is this correct?

Reply:
I think the templating system requires everything to be a string but will test to make sure

Reply:
Oh maybe I'm wrong... I see we're passing ints into source_type_id. Will test to make sure... it's possible both will work.

Reply:
But I bet it's because `str(45)` == '45' (which is valid JSON) and `str(False)` == `False` (which is not.)

Reply:
Just confirmed my suspicions via manually testing: the templating function in this stuff appears to just call `str()` on whatever you give it. So you can pass ints and strings but not bools as template variables.

Reply:
Tested locally and got `"auto_install": true,`. Thanks!